src/components/SearchBar.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { ReactNode } from 'react';
+import {
+  isSearchBarAvailableForCurrentPlatform,
+  SearchBarCommands,
+  SearchBarProps,
+} from 'react-native-screens';
+import { View } from 'react-native';
+
+// Native components
+import SearchBarNativeComponent, {
+  Commands as SearchBarNativeCommands,
+} from '../fabric/SearchBarNativeComponent';
+
+export const NativeSearchBar: React.ComponentType<SearchBarProps> &
+  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+export const NativeSearchBarCommands: SearchBarCommandsType =
+  SearchBarNativeCommands as any;
+
+type SearchBarCommandsType = {
+  blur: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  focus: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  clearText: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  toggleCancelButton: (
+    viewRef: React.ElementRef<typeof NativeSearchBar>,
+    flag: boolean
+  ) => void;
+  setText: (
+    viewRef: React.ElementRef<typeof NativeSearchBar>,
+    text: string
+  ) => void;
+};
+
+class SearchBar extends React.Component<SearchBarProps> {
+  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
+
+  constructor(props: SearchBarProps) {
+    super(props);
+    this.nativeSearchBarRef = React.createRef();
+  }
+
+  _callMethodWithRef(method: (ref: SearchBarCommands) => void) {
+    const ref = this.nativeSearchBarRef.current;
+    if (ref) {
+      method(ref);
+    } else {
+      console.warn(
+        'Reference to native search bar component has not been updated yet'
+      );
+    }
+  }
+
+  blur() {
+    this._callMethodWithRef(ref => NativeSearchBarCommands.blur(ref));
+  }
+
+  focus() {
+    this._callMethodWithRef(ref => NativeSearchBarCommands.focus(ref));
+  }
+
+  toggleCancelButton(flag: boolean) {
+    this._callMethodWithRef(ref =>
+      NativeSearchBarCommands.toggleCancelButton(ref, flag)
+    );
+  }
+
+  clearText() {
+    this._callMethodWithRef(ref => NativeSearchBarCommands.clearText(ref));
+  }
+
+  setText(text: string) {
+    this._callMethodWithRef(ref => NativeSearchBarCommands.setText(ref, text));
+  }
+
+  render() {
+    if (!isSearchBarAvailableForCurrentPlatform) {
```

Comment:
Did you maybe implement `SearchBar` for windows @marlenecota?

Reply:
I'm not very familiar with this module, but I believe it was a partial implementation that did not include SearchBar.